app/views/get-started/deploy-your-prototype/index.html

```diff
@@ -2,8 +2,8 @@
 
 {% set sectionId = "get-started" %}
 {% set sectionTitle = "How to guides" %}
-{% set pageId = "deploy-your-prototype" %}
-{% set pageTitle = "Deploy your prototype" %}
+{% set pageId = "deploying-your-prototype" %}
+{% set pageTitle = "Deploying your prototype" %}
 
 {% block subNav %}
 {% include "../navigation.html" %}
```

Comment:
`navigation.html` needs to be updated to reflect the pageId of "deploying-your-prototype" and pageTitle of "Deploying your prototype".

Reply:
You also need to rename the directory to "deploying-your-prototype"